api/turing/service/router_deployment_service.go

```diff
@@ -563,27 +556,13 @@ func deleteSecret(controller cluster.Controller, secret *cluster.Secret, isClean
 	return controller.DeleteSecret(context.Background(), secret.Name, secret.Namespace, isCleanUp)
 }
 
-func createPVC(
-	ctx context.Context,
-	controller cluster.Controller,
-	namespace string,
-	pvc *cluster.PersistentVolumeClaim,
-) error {
-	select {
-	case <-ctx.Done():
-		return errors.New("timeout deploying service")
-	default:
-		return controller.ApplyPersistentVolumeClaim(ctx, namespace, pvc)
-	}
-}
-
-func deletePVC(
+func deleteStatefulSetPVCs(
 	controller cluster.Controller,
 	namespace string,
-	pvc *cluster.PersistentVolumeClaim,
+	statefulSetName string,
 	isCleanUp bool,
 ) error {
-	return controller.DeletePersistentVolumeClaim(context.Background(), pvc.Name, namespace, isCleanUp)
+	return controller.DeleteStatefulSetPersistentVolumeClaims(context.Background(), statefulSetName, namespace, isCleanUp)
```

Comment:
nit: would it be better to just use `ignoreNotFound` as per signature instead of `isCleanUp`?

Reply:
`isCleanUp` is like the standard name of the such a `ignoreNotFound` flag passed around everywhere by the router deployment service though, so I just used the same name to keep things the same but I can refactor all of them too to use `ignoreNotFound`.

Reply:
nah no strong opinion

Reply:
Okay then I'll leave it as is then.